source/_integrations/drop_connect.markdown

```diff
@@ -60,7 +49,19 @@ To enable the DROP integration, use the DROP Connect app to connect your DROP Hu
 - Enter the MQTT broker address, port number, username, and password.
 - Click **Connect** and confirm that the DROP Hub has connected to the MQTT broker.
 
-Once the DROP Hub is connected to your MQTT broker, the devices on your DROP system will be discovered by Home
+Once the DROP Hub is connected to your MQTT broker, the devices on your DROP system should be discovered by Home
 Assistant.
-Go to **Settings > Devices & Services** in the Home Assistant UI to add your DROP devices.
 
+{% details "Manual configuration steps" %}
+
+- Browse to your Home Assistant instance.
+- Go to **{% my integrations title="Settings > Devices & Services" %}**.
+- Set up the new discovered devices.
+
+{% enddetails %}
```

Comment:
This needs to include our config_flow.md template file

Reply:
This integration is not set up by adding a config entry manually. Devices need to be discovered, and after that they can be set up. By adding the `config_flow.md` template the wrong instructions are shown.

Reply:
I ended using part of the template code.

Reply:
@frenck may be you can have another look as using `config_flow.md` is not appropriate here.